apps/tlon-mobile/src/components/AuthenticatedApp.tsx

```diff
@@ -15,6 +15,7 @@ import { AppStateStatus } from 'react-native';
 
 import { useCheckAppUpdated } from '../hooks/analytics';
 import { useDeepLinkListener } from '../hooks/useDeepLinkListener';
+import useFindContactSuggestions from '../hooks/useFindContactSuggestions';
```

Comment:
Would it be possible to do this initialization in the `app` or `shared` package? If not we should duplicate the logic in web.